### PR TITLE
bt-157: add file extension

### LINKED ISSUE #204: Fix file extension in shared folder for auth constants

### PR DESCRIPTION
**Pull request by mskmee**

### CODE REVIEW

File: shared/src/bundles/users/validation-schemas/user-sign-in.validation-schema.ts
Comment (nikita-remeslov) [inline shared/src/bundles/users/validation-schemas/user-sign-in.validation-schema.ts:1]: create a folder for this file 'constants' - shared/src/bundles/users/validation-schemas/auth-constants.ts

also, rename it to `constants.ts`

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

File: shared/src/bundles/users/validation-schemas/constants/constants.ts
Comment (nikita-remeslov) [inline shared/src/bundles/users/validation-schemas/constants/constants.ts:1]: `AUTH_CONSTANTS` better name

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Conflicts

File: shared/src/bundles/users/validation-schemas/constants/constants.ts
Comment (mskmee) [reply shared/src/bundles/users/validation-schemas/constants/constants.ts:1]: fix

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
if you want to keep your file name.

1. Rename auth.contants.ts
2. Create root file constants.ts

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**